utils/ssh.py

```diff
@@ -309,13 +309,13 @@ def cpu_spike(self, seconds=60, cpus=2, **kwargs):
 
     def run_rails_command(self, command, timeout=RUNCMD_TIMEOUT, **kwargs):
         logger.info("Running rails command %r", command)
-        return self.run_command('/var/www/miq/vmdb/bin/rails runner {command}'.format(
+        return self.run_command('cd /var/www/miq/vmdb; bin/rails runner {command}'.format(
```

Comment:
This is required as when the absolute path is used it will not take the installed gem into the account.